nrf_security/doc/backend_config.rst

```diff
@@ -244,10 +47,11 @@ AES core support can be enabled by setting the :option:`CONFIG_MBEDTLS_AES_C` Kc
 +--------------+-----------------------------------+
 
 .. note::
-   * The `Arm CryptoCell cc310 backend`_ is limited to key sizes of 128 bits.
+   * The :ref:`nrf_security_backends_cc_310` is limited to key sizes of 128 bits.
```

Comment:
There is no use of the abbreviated form that uses a split betwen cc and 310. 
Please call this cc310 in the link

Reply:
This is now fixed everywhere.